java/vector/src/main/java/org/apache/arrow/vector/file/json/JsonFileReader.java

```diff
@@ -260,10 +260,10 @@ private void readVector(Field field, FieldVector vector) throws JsonParseExcepti
           ((ListVector) vector).getMutator().setLastSet(count);
           break;
         case VARBINARY:
-          ((NullableVarBinaryVector) vector).getMutator().setLastSet(count - 1);
+          ((NullableVarBinaryVector)vector).setLastSet(count - 1);
```

Comment:
Nit:
Should we add whitespace after closing `)` in casting? This is specified in 
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
item 5